app/src/main/java/com/github/polypoly/app/MapActivity.kt

```
@@ -59,6 +63,12 @@ class MapActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        val db = Firebase.database
+        try {
+            db.setPersistenceEnabled(false) // disable local caching and do only write-through instead
+        } catch(_: java.lang.Exception) { }
```

Ignoring all exceptions deserves a comment :)